src/main/java/com/flansmod/common/network/PacketModConfig.java

```diff
@@ -45,9 +51,16 @@ public class PacketModConfig extends PacketBase {
     public float nameTagRenderRange;
     public float nameTagSneakRenderRange;
     public float maxHealth;
+<<<<<<< Updated upstream
     public int bonusRegenTickDelay;
     public int bonusRegenFoodLimit;
     public float bonusRegenAmount;
+=======
+    
+    public boolean enableBlockPenetration;
+    public float masterBlockPenetrationModifier;
+    public String[] penetrableBlocksArray;
+>>>>>>> Stashed changes
```

Comment:
These lines are from git when it hits a conflict on merge and should be removed. Looks like you can get away with just removing these to resolve the conflict although sometimes you may have to change some stuff around when you run into this.